pylua/timer.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+# timer works with milliseconds
+
+import threading
+
+
+def convert_milliseconds_to_seconds(milliseconds):
+    milliseconds_in_second = 1000.0
+    return milliseconds/milliseconds_in_second
+
+
+def convert_seconds_to_milliseconds(seconds):
+    milliseconds_in_second = 1000.0
+    return seconds*milliseconds_in_second
+
+
+class PyTimer(object):
+
+    def __init__(self, task, interval):
+        self._task = task
+        self._interval = convert_milliseconds_to_seconds(interval)
+        self._timer = None
+
+    def start(self, interval=None):
+        if interval is not None:
+            self._interval = float(convert_milliseconds_to_seconds(interval))
+        self.create()
+        self._timer.start()
+
+    def create(self):
+        self._timer = threading.Timer(self._interval, self._task)
+
+    def stop(self):
+        self._timer.cancel()
+
+    def reset(self):
+        self._timer.cancel()
+        self.create()
+        self._timer.start()
+
+    @property
+    def interval(self):
```

Comment:
@dtrunov, why you need to expose interval propose? What are use cases? On my opinion this property and parameter in start are redundant. The possibility to change interval makes logic more complicated.